### PR TITLE
[SIM-3111] - Add searchable select filters where needed

### PR DESCRIPTION
**Pull request by avenderossa**
Updating searchable select assigns changed to update the disabled and placeholder values if these have changed

### CODE REVIEW

File: lib/searchable_select.ex [hunk 1]
```diff
@@ -47,6 +47,8 @@ defmodule SearchableSelect do
     socket =
       socket
       |> assign(:search, "")
+      |> assign(:disabled, assigns[:disabled] || false)
+      |> assign(:placeholder, assigns[:placeholder] || "Search")
```
Comment (Tom-Wyszkowski) [inline lib/searchable_select.ex:51]: Already defaulted in the initial update function. A good use case for `assign_new`

Comment (louisvisser) [inline lib/searchable_select.ex:51]: No need for defaults at all here methinks - on 1st render they were already defined as part of socket.assigns, and defaults were given too (lines 64 & 74). Tested this and seems to work right for me:
```suggestion
      |> assign(:disabled, assigns[:disabled])
      |> assign(:placeholder, assigns[:placeholder])
```

**Review by louisvisser — COMMENTED** [review]
Nice one! Just a comment regarding defaults:

File: lib/searchable_select.ex (hunk 1 above)
Comment (Tom-Wyszkowski) [reply lib/searchable_select.ex:51]: Agreed

**Pull request merged**